clusterloader2/testing/load/config.yaml

```diff
@@ -57,15 +57,27 @@
 
 # BEGIN scheduler-throughput section
 # TODO( https://github.com/kubernetes/perf-tests/issues/1027): Lower the number of "min-pods" once we fix the scheduler throughput measurement.
-{{$totalSchedulerThroughputPods := MaxInt (MultiplyInt 2 $MIN_PODS_IN_SMALL_CLUSTERS) .Nodes}}
-{{$schedulerThroughputPodsPerDeployment := .Nodes}}
-{{$schedulerThroughputNamespaces := DivideInt $totalSchedulerThroughputPods $schedulerThroughputPodsPerDeployment}}
+{{$schedulerThroughputReplicasPerNamespace := 0}}
+{{$schedulerThroughputPodsPerDeployment := 0}}
+{{$schedulerThroughputNamespaces := 0}}
+{{if $IS_SMALL_CLUSTER}}
+  {{$schedulerThroughputReplicasPerNamespace = 1}}
+  {{$schedulerThroughputPodsPerDeployment = .Nodes}}
+  {{$schedulerThroughputNamespaces = 1}}
+{{end}}
+{{if and (ge .Nodes 100) (lt .Nodes 1000)}}
```

Comment:
This duplicates `if $IS_SMALL_CLUSTER` and `if ge .Nodes 1000`.

Could you deduplicate gating logic using `else` ?

Do we even need 3 cases. I think <100 and 100>= should be enough with previous version of calculating totalPods.

Reply:
Simplified the logic down to:
```
{{$totalSchedulerThroughputPods := MaxInt (MultiplyInt 2 $MIN_PODS_IN_SMALL_CLUSTERS) .Nodes}}
{{$schedulerThroughputReplicasPerNamespace := 1}}
{{$schedulerThroughputPodsPerDeployment := .Nodes}}
{{$schedulerThroughputNamespaces := DivideInt $totalSchedulerThroughputPods $schedulerThroughputPodsPerDeployment}}

{{$schedulerThroughputNamespaces := IfThenElse $IS_SMALL_CLUSTER 1 $schedulerThroughputNamespaces}}

{{if $IS_LARGE_CLUSTER}}
  {{$schedulerThroughputReplicasPerNamespace = DivideInt .Nodes $LARGE_CLUSTER_NODES}}
  {{$schedulerThroughputPodsPerDeployment = DivideInt .Nodes $schedulerThroughputReplicasPerNamespace}}
  {{$schedulerThroughputNamespaces = 1}}
{{end}}
```

Reply:
We could do one more polish here with:
```
{{$schedulerThroughputNamespaces := IfThenElse (or $IS_SMALL_CLUSTER $IS_LARGE_CLUSTER) 1 $schedulerThroughputNamespaces}}
```
Please let me know what is your opinion

Reply:
It is 1 in both cases but the cause is different. I do not think this would increase readability.

Reply:
Ack